### DISCUSSION TITLE
code-server command not working after installing zsh

### DISCUSSION THREAD

**Discussion by msjahid** [coder/code-server#1948] [Q&A] [answered]
When my terminal was the default shell then code-server command was working. But suddenly I have been installed zsh in my pc so I change from default to zsh. Presently code-server command not working. When I went to install it again, The system called that code-server is already installed. 
![Screenshot_20200807_151214](https://user-images.githubusercontent.com/12425488/89630836-b7abd180-d8c1-11ea-8b44-ecbac6744a5d.png)

Now, I can not open code-server. Have any solution?
Thank you

**Comment by code-asher** [comment, marked as answer]
This probably means you need to add `~/.local/bin` to your `PATH`.

**Comment by code-asher** [comment]
You can also try running it directly: `~/.local/bin/code-server`

**Comment by msjahid** [comment]
Thank you so much.
